packages/babel-cli/src/babel/dir.js

```diff
@@ -125,10 +129,17 @@ export default function(commander, filenames, opts) {
     const filename = filenames[index];
 
     handle(filename, function(err) {
-      if (err) throw err;
+      if (err) throw Error(err);
```

Comment:
this probably should be a `new Error`

Reply:
they're equivalent https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error

Reply:
While they might be equivalent, it's most commonly used as a constructor - I'd prefer to `new` it for consistency.